IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormCommitTests.cs

```diff
@@ -483,6 +483,9 @@ private void RunFormTest(Func<FormCommit, Task> testDriverAsync, CommitKind comm
                         CommitKind.Fixup => _commands.StartFixupCommitDialog(owner: null, _referenceRepository.Module.GetRevision()),
                         _ => throw new ArgumentException($"Unsupported commit kind: {commitKind}", nameof(commitKind))
                     });
+
+                    // Await updated FileViewer
+                    ThreadHelper.JoinPendingOperations();
                 },
                 testDriverAsync);
         }
```

Comment:
The call of `JoinPendingOperations` might be too early in `showForm:` because e.g. `OnShown` might be processed later. This depends on how the specific Form works.
Let's see whether the tests pass stably. If not, this can be moved into the `runTestAsync:` with a follow-up PR, and - if necessary - even with a loop checking whether the `FileViewer` has been populated finally.